XCUITests/FindInPageTest.swift

```diff
@@ -140,7 +140,12 @@ class FindInPageTests: BaseTestCase {
         navigator.goto(TabTray)
 
         waitForExistence(app.cells.staticTexts["The Book of Mozilla"])
-        app.cells.staticTexts["The Book of Mozilla"].tap()
+        //app.cells.staticTexts["The Book of Mozilla"].tap()
+        if iPad() {
+            app.collectionViews.cells["The Book of Mozilla. Currently selected tab."].tap()
```

Comment:
I would try to use other option if exists to not use `.Currently selected tab` in the cell name... For example, this would work too:
`app.cells.staticTexts["The Book of Mozilla"].firstMatch.tap()`

In fact, that line works for both iPhone and iPad, you would not have to add the device condition